build/dependencies.props

```diff
@@ -5,7 +5,7 @@
     <AppInsightsVersion>2.1.0-beta3</AppInsightsVersion>
```

Comment:
I think the test project wouldn't be able to compile without the common.props change to add NS.Library.NETFramework.